transfermethodui/src/test/java/com/hyperwallet/android/ui/transfermethod/view/widget/AbstractMaskedInputWidgetTest.java

```diff
@@ -0,0 +1,95 @@
+package com.hyperwallet.android.ui.transfermethod.view.widget;
```

Comment:
I'm not sure if this test should be here. I think it should be under the androidTest folder. Let's discuss this offline

Reply:
This is just a roboelectric test though? (actually, Roboelectric isn't even needed for this)